### ISSUE TITLE
[Discussion] loss function of finetune llava 1.5 with sft 

### ISSUE THREAD

**Issue by bollossom** [haotian-liu/LLaVA#1782]
### Discussion

![屏幕截图 2024-11-25 095623](https://github.com/user-attachments/assets/8f7b78e1-20cd-40e5-a323-0f9c8bd1d70c)
Is it normal for loss to be so shaky?
